src/npm/hook.js

```diff
@@ -29,38 +29,31 @@ import { generateDataFromMeasures } from '../shared/generate'
 
   NOTE: This should only be used in development mode.
 */
-const registerObserver = (params, callback) => {
-  params = params || {}
-
-  // TODO: Is there any way to polyfill this API ?
```

Comment:
this is a devtool for chrome + firefox. no need to polyfill.
Writing add-ons/extensions is a whole different problem

Reply:
see #52, we'll polyfill for older browsers if we somewhen find a proper polyfill for the API :)